win32ss/gdi/ntgdi/freetype.c

```diff
@@ -2821,7 +2821,9 @@ GetFontFamilyInfoForSubstitutes(const LOGFONTW *LogFont,
         GetFontFamilyInfoForList(&lf, Info, pFromW->Buffer, pCount, MaxCount,
                                  &Win32Process->PrivateFontListHead);
         IntUnLockProcessPrivateFonts(Win32Process);
-        break;
+
+        if (LogFont->lfFaceName[0])
```

Comment:
Maybe adding a comment why you stop the enumeration loop only if *lfFaceName is not NULL could be nice.

Reply:
OK. Comment added.